libs/cardano-ledger-core/src/Cardano/Ledger/Core.hs

```diff
@@ -152,6 +152,16 @@ class
   -- | For end use by eg. diffusion layer in transaction submission protocol
   wireSizeTxF :: SimpleGetter (Tx era) Word32
 
+  -- | For fee calculation and estimations of impact on block space
+  -- To replace `sizeTxF` after it has been proved equivalent to it .
+  sizeTxForFeeCalculation :: SafeToHash (TxWits era) => Tx era -> Integer
+  sizeTxForFeeCalculation tx =
+    fromIntegral $
+      originalBytesSize (tx ^. bodyTxL)
+        + originalBytesSize (tx ^. witsTxL)
+        + fromSMaybe 1 (originalBytesSize <$> (tx ^. auxDataTxL))
```

Comment:
If `hlint` knew about `StrictMaybe` I think it would suggest this:

```haskell
strictMaybe 1 originalBytesSize (tx ^. auxDataTxL)
```

Reply:
thanks!